presto-main/src/main/java/com/facebook/presto/sql/planner/BasePlanFragmenter.java

```diff
@@ -181,6 +181,27 @@ private SubPlan buildFragment(PlanNode root, FragmentProperties properties, Plan
 
         // Only delegate non-coordinatorOnly plan fragment to native engine
         if (isNativeExecutionEnabled(session) && !properties.getPartitioningHandle().isCoordinatorOnly()) {
+            if (root instanceof OutputNode) {
+                // OutputNode is special in that it can have duplicate output variables. Regular plan nodes,
```

Comment:
can you add to this comment that the OutputNode variables aren't really used, to explain why it's okay for correctness.  Also maybe note why OutputNodes are special (as i understand, it's the PhysicalOperations that can't have duplicates, and OutputNode doesn't get translated to a PhysicalOperation, but NativeExecutionNode does).

Reply:
@rschlussel Rebecca, I tried expanding the comment, but couldn't figure out wording that would make sense. Here is what I have now:

```
                // OutputNode is special in that it can have duplicate output variables. Regular plan nodes,
                // including NativeExecutionNode, must have unique output variables.
                // Check if OutputNode has duplicate output variables and remove these.
```

Any chance you could suggest a specific edit?

Reply:
What do you think of the below           
``` 
               // OutputNode is special in that it can have duplicate output variables since it
               // does not get converted to a PhysicalOperation for execution.
               // Regular plan nodes, including NativeExecutionNode, must have unique output variables.
               // Check if OutputNode has duplicate output variables and remove these.
               // This is safe because OutputNode variables will not be used by the workers.

```

Reply:
@rschlussel Thank you, Rebecca. Let me use that.